alkira/resource_alkira_credential_fortinet_instance.go

```diff
@@ -25,9 +28,22 @@ func resourceAlkiraCredentialFortinetInstance() *schema.Resource {
 				Required:    true,
 			},
 			"license_key": &schema.Schema{
-				Description: "Fortinet license key.",
-				Type:        schema.TypeString,
-				Optional:    true,
+				Description: "Fortinet license key is treated as a file path by default. `license_key` " +
```

Comment:
Why just have a new optional field as `license_key_file`? Usually, we want identifier to be single purpose in HCL.